### ISSUE TITLE
Adding ability for var to assign to attributes?

### ISSUE THREAD

**Issue by kirbyfan64** [arguiot/DisplayJS#37]
I mean something like this:

```html
<a var:href="url"></a>
```

so that the `url` variable gets assigned to the `href` attribute.

**Comment by arguiot** [comment]
Yeah, it would be a good idea💡, I'm busy with my other projects, but I'll try to do it. You can submit a PR if you want this feature more quickly.

**Comment by arguiot** [comment]
Hello, it's been a long time... 😅. Please give me all attributes you were thinking about, inorder to put them in DisplayJS.
